Package.swift

```diff
@@ -36,13 +36,22 @@ let package = Package(
     targets: [
         .target(name: "RevenueCat",
                 path: "Sources",
-                exclude: ["Info.plist", "LocalReceiptParsing/ReceiptParser-only-files"]),
+                exclude: ["Info.plist", "LocalReceiptParsing/ReceiptParser-only-files"],
+                resources: [
+                    .copy("../Sources/PrivacyInfo.xcprivacy")
```

Comment:
This kind of breaks (cc @neonacho)
![Screenshot 2023-06-09 at 15 40 13](https://github.com/RevenueCat/purchases-ios/assets/685609/61937884-16be-4098-8a05-b0cf61eb931f)

/cc @

Reply:
I'll leave this for now since it's the only way to add it.